docs/resources.md

```diff
@@ -0,0 +1,39 @@
+# Resources
+
+The following are resources about Melange collected around the web.
+
+## Blog posts, podcasts, talks
+
+| Date | Type | Description |
+|----- | ---- | ----------- |
+| Sep 2024 | talk | [Type-Safe React with OCaml by Dillon Mulroy](https://reactalicante.es/#schedule) (video not yet uploaded) |
+| Sep 2024 | podcast | [The State of Full-Stack OCaml (with António Monteiro)](https://pod.link/developer-voices/episode/4666dc4d7aa59599852c02b6415a36d4) ([video](https://www.youtube.com/watch?v=vSSpbAI_-t0))
+| May 2024 | blog | [Melange 4.0 is here](https://melange.re/blog/posts/melange-4-is-here)
```

Comment:
This is a bit self-referencial, idk how i feel about having so many entries (more than 50%) being from the official blog.

Maybe it could be listed just once? e.g.

| Date | Type | Description |
|----- | ---- | ----------- |
| --- | blog | [Official Melange blog](https://melange.re/blog/)

cc @anmonteiro

Reply:
I was wondering if it made sense to add everything I could find. Maybe it makes sense to apply some curation. I see two ways to do it:

1. Add a Most Recommended (or Favorites?) section at the top and include the best stuff there, then the blog post/talks/podcasts section includes everything
2. Pare this section down to only include release announcements from official blog

Wdyt?

Reply:
I would keep our own blog outside of the "Resources" tbh